flink-clients/src/test/java/org/apache/flink/client/CliFrontendInfoTest.java

```diff
@@ -61,7 +61,7 @@ public void testShowExecutionPlan() {
 		replaceStdOut();
 		try {
 
-			String[] parameters = new String[] { CliFrontendTestUtils.getTestJarPath() };
+			String[] parameters = new String[] { CliFrontendTestUtils.getTestJarPath(), "-v", "true", "/dev/null", "/dev/null"};
```

Comment:
`-v` is a valid option. I would change it to `-f` which is also the option initially causing the problem.